packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -423,6 +426,7 @@ class _CodeViewState extends State<CodeView>
       FileSearchField(
         debuggerController: widget.controller,
       ),
+      width: extraWideSearchTextWidth,
```

Comment:
what effect does specifying this width and the width for the container achieve?

Reply:
Hmm I'm confused - this width is the width for the container (in`wrapInElevatedCard`)

Reply:
What confuses me is its child also has a width (line 416). I'd hope you could just specify one width for the outer parent and have all widths be computed based on that.

Reply:
you may need to expand out the few lines that are omitted in the git diff. This line (429) is part of the `buildFileSearchField` method, which calls `wrapInElevatedCard` with the parameter `width: extraWideSearchTextWidth,`.  The above use of `width` on line 416, is part of the `wrapInElevatedCard` method.